grin.toml

```diff
@@ -73,7 +73,7 @@ run_test_miner = false
 #test miner wallet URL (burns if this doesn't exist)
 #test_miner_wallet_url = "http://127.0.0.1:13415"
 
-[server.dandelion_config]
+#[server.dandelion_config]
```

Comment:
Is this necessary to be commented out?
Feels like it will bite someone if they attempt to change a value below and don't uncomment this.

Reply:
I don't know why but if it's uncommented it will ask for the other values when doing ```grin server run```. I will take a look though.

Reply:
Actually should be allowed in the toml file itself.
So either our code or the serde parsing code that is causing problems.

> Empty tables are allowed and simply have no key/value pairs within them.

https://github.com/toml-lang/toml#table

Reply:
I'd leave the [server.dandelion_config] section in there uncommented and required, but then allow for the values themselves to be optional.